beacon-chain/powchain/service.go

```diff
@@ -225,6 +246,7 @@ func (w *Web3Service) ProcessLog(VRClog gethTypes.Log) {
 // the ETH1.0 chain by trying to ascertain which participant deposited
 // in the contract.
 func (w *Web3Service) ProcessDepositLog(VRClog gethTypes.Log) {
+	totalDepositsCount.Inc()
```

Comment:
the `Inc()` method is to increment this , right ?

Reply:
Yep!